CHANGELOG.md

```diff
@@ -3083,6 +3123,20 @@ Additions to existing modules
   prependVLams : List String → Term → Term
   ```
 
+* Added new definitions to `Relation.Binary.Bundles`:
```

Comment:
This entry isn't needed as the full details should be included in the main `breaking` change entry above. Anything that you think should be preserved in this entry should be moved to the above entry.

Reply:
OK